### ISSUE TITLE
Github Action crashes

### ISSUE THREAD

**Issue by mschfh** [stepci/stepci#180]
### What happened?

The action crashes with a `TypeError`

### What did you expect to happen?

The action doesn't crash with a `TypeError` and completes.

### Version

latest

### Environment

Node.js v20.9.0

### How can we reproduce this bug?

```
  api_test:
    name: API Tests
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
      - name: Step CI Action
        uses: stepci/stepci@e4043bfde104e2f4b3a44fa48d5fdb6793d8664a # v2.7.0
        with:
          workflow: "tests/workflow.yml"
          envs: "myvar=foo othervar=bar"
        env:
          STEPCI_DISABLE_ANALYTICS: true
```

### Relevant log output

```shell
/usr/local/lib/node_modules/stepci/node_modules/@stepci/runner/dist/index.js:89
    const concurrency = options?.concurrency || workflow.config?.concurrency || Object.keys(workflow.tests).length;
                                                                                       ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at run (/usr/local/lib/node_modules/stepci/node_modules/@stepci/runner/dist/index.js:89:88)
    at runFromYAML (/usr/local/lib/node_modules/stepci/node_modules/@stepci/runner/dist/index.js:64:12)
```

### Would you be interested in working on a bugfix for this issue?

- [ ] Yes! Assign me

**Comment by mishushakov** [comment]
Fixed in 1.11.2

**Comment by mishushakov** [comment]
Ps. If anyone else is having the issue you should migrate from using `include` in your workflow file to use the new `$ref` syntax instead

https://docs.stepci.com/guides/organising-workflows.html
